pwa/app/lib/utils.ts

```diff
@@ -28,6 +32,14 @@ export function sortMatchComparator(a: Match, b: Match) {
   return a.set_number - b.set_number || a.match_number - b.match_number;
 }
 
+export function sortAwardsComparator(a: Award, b: Award) {
```

Comment:
`src/backend/common/consts/award_type.py` specifies an additional award sort order. We could copy that here.